### PR TITLE
WIP: implement quantile loss function

### PR DESCRIPTION
**Pull request by trdvangraft**
Add quantile loss to fastai, solves #3279

### CODE REVIEW

**Comment by ReinierKoops** [comment]
Something seems to have gone wrong, maybe new pull request?

**Comment by ReinierKoops** [comment]
Ah, now I see. The whole library had to be rebuild because of the addition of your one feature.

**Comment by hamelsmu** [comment]
@ReinierKoops I converted this to a draft PR while you sort the conflicts (and also b/c the PR title says WIP)

**Comment by jph00** [comment]
@ReinierKoops please let me know when this is ready to review.

**Comment by jph00** [comment]
@trdvangraft please let me know when this is ready to review

**Comment by tcapelle** [comment]
Would be cool to do something like this: https://www.angioi.com/time-series-encoder-decoder-tensorflow/ using the new `GaussianLoss`, basically, instead of doing a regression, you predict mean and std and fit a normal distribution that minimizes NLL. I can help you with this if you want.

**Comment by trdvangraft** [comment]
Hey sorry for the long radio silence, I got tangled up in some other work. I think we can package this MR together @tcapelle to add some more loss functions. But first we will have to figure out how to get rid off the notebook duplication and get it updated with master again.

**Comment by tcapelle** [comment]
Just create a new PR
